src/data.yaml

```diff
@@ -1,4 +1,30 @@
 jobs:
+- name: Metabolic
+    website: https://www.metabolic.nl
+    jobs: https://www.metabolic.nl
+    description: Sustainability consulting
+    remote: true
+    speculative: false
+    field: Misc.
+    geo:
+      - country: Netherlands
+        lat: 52.399131
+        long: 4.909506
+    review: https://www.glassdoor.com/Overview/Working-at-Metabolic-EI_IE1257662.11,20.htm
```

Comment:
This URL needs to point to the Reviews page, otherwise the crawler can't find the rating.